db/migration/0050_tagbot_entitlement.sql

```diff
@@ -0,0 +1,24 @@
+--   Copyright 2020 MSolution.IO
+--
+--   Licensed under the Apache License, Version 2.0 (the "License");
+--   you may not use this file except in compliance with the License.
+--   You may obtain a copy of the License at
+--
+--       http://www.apache.org/licenses/LICENSE-2.0
+--
+--   Unless required by applicable law or agreed to in writing, software
+--   distributed under the License is distributed on an "AS IS" BASIS,
+--   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--   See the License for the specific language governing permissions and
+--   limitations under the License.
+
+CREATE TABLE user_tagbot (
```

Comment:
Prefixing tagbot's tables by `tagbot_` seems better to me
So here it should be `tagbot_user`